### DISCUSSION TITLE
Healthchecking backends

### DISCUSSION THREAD

**Discussion by adameasten** [dotnet/yarp#224] [General] [answered]
Hello, i cant figure out how to enable backendprobes to keep track of the health on my backends! Is this the right forum to ask for advice or for examples on how it works? :)

**Comment by Tratcher** [comment, marked as answer]
This is the right place. Unfortunately the health checks aren't fully wired up yet. I've filed https://github.com/microsoft/reverse-proxy/issues/228 to finish that.
